packages/agent-toolkit/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@clerk/agent-toolkit",
   "version": "0.0.5",
-  "private": true,
```

Comment:
This change will release the package, right?

Reply:
yup